model.py

```diff
@@ -1,7 +1,6 @@
 import tensorflow as tf
+from utils import Screenshot
 
-IMG_W = 200
-IMG_H = 66
 OUT_SHAPE = 5
```

Comment:
just commenting out of interest - This constant probably also belongs to a utils object. Since the utils defines the controller and the model should just react to that. This might come into play if someone adds more buttons or extends this to a different game.